src/main/resources/config/en.yml

```diff
@@ -76,10 +76,15 @@ Experiment_JdbcUsername: "username"
 Experiment_JdbcPassword: "password"
 
 # Webhook Delivery
+# Whether to use webhooks to send chat messages to Discord
```

Comment:
Should state what webhooks do (eg. 'uses the player's name and skin's head as the username and avatar in Discord')